src/components/tables/CellProgressBar.js

```diff
@@ -24,7 +24,7 @@ export const CellProgressBar = ({ value, reverse = false }) => {
       color = 'success'
     }
   }
-  if (value) {
+  if (value > 0) {
```

Comment:
This was needed because value can now be 2 decimal places.

So whilst 0 == false, 0.00 != false thus 0.00 was being treated as a value, although now I reflect on it in hindsight, 0% is a value right? So shouldn't we be showing an empty percentage bar for 0% instead of "No data" 🤔